docs/src/test/count_outfiles.ok

```diff
@@ -0,0 +1,170 @@
+File linked successfully.
+File linked successfully.
+ERROR: README.md not found in ../src/dpo
+File linked successfully.
+ERROR: README.md not found in ../src/dst
+File linked successfully.
+File linked successfully.
+File linked successfully.
+File linked successfully.
+File linked successfully.
+File linked successfully.
+File linked successfully.
+File linked successfully.
+File linked successfully.
+File linked successfully.
+File linked successfully.
+File linked successfully.
+File linked successfully.
+File linked successfully.
+File linked successfully.
+File linked successfully.
+File linked successfully.
+File linked successfully.
+File linked successfully.
+File linked successfully.
+File linked successfully.
+ant.md
+Names: 1,        Desc: 1,        Syn: 1,        Options: 1,        Args: 1
+cts.md
+Names: 3,        Desc: 3,        Syn: 3,        Options: 3,        Args: 3
+./md/man2/dbSta.md doesn't exist. Continuing
+dft.md
+Names: 4,        Desc: 4,        Syn: 4,        Options: 4,        Args: 4
+dpl.md
+Names: 6,        Desc: 6,        Syn: 6,        Options: 6,        Args: 6
+./md/man2/dpo.md doesn't exist. Continuing
+drt.md
+Names: 3,        Desc: 3,        Syn: 3,        Options: 3,        Args: 3
+./md/man2/dst.md doesn't exist. Continuing
+fin.md
+Names: 1,        Desc: 1,        Syn: 1,        Options: 1,        Args: 1
+gpl.md
+Names: 2,        Desc: 2,        Syn: 2,        Options: 2,        Args: 2
+grt.md
+Names: 13,        Desc: 13,        Syn: 13,        Options: 13,        Args: 13
+gui.md
+Names: 46,        Desc: 46,        Syn: 46,        Options: 46,        Args: 46
+ifp.md
+Names: 3,        Desc: 3,        Syn: 3,        Options: 3,        Args: 3
+mpl.md
+Names: 1,        Desc: 1,        Syn: 1,        Options: 1,        Args: 1
+mpl2.md
+Names: 2,        Desc: 2,        Syn: 2,        Options: 2,        Args: 2
+pad.md
+Names: 14,        Desc: 14,        Syn: 14,        Options: 14,        Args: 14
+par.md
+Names: 6,        Desc: 6,        Syn: 6,        Options: 6,        Args: 6
+pdn.md
+Names: 9,        Desc: 9,        Syn: 9,        Options: 9,        Args: 9
+ppl.md
+Names: 11,        Desc: 11,        Syn: 11,        Options: 11,        Args: 11
+psm.md
+Names: 4,        Desc: 4,        Syn: 4,        Options: 4,        Args: 4
+rcx.md
+Names: 9,        Desc: 9,        Syn: 9,        Options: 9,        Args: 9
+rmp.md
+Names: 1,        Desc: 1,        Syn: 1,        Options: 1,        Args: 1
+rsz.md
+Names: 17,        Desc: 17,        Syn: 17,        Options: 17,        Args: 17
+stt.md
+Names: 1,        Desc: 1,        Syn: 1,        Options: 1,        Args: 1
+tap.md
+Names: 5,        Desc: 5,        Syn: 5,        Options: 5,        Args: 5
+upf.md
+Names: 11,        Desc: 11,        Syn: 11,        Options: 11,        Args: 11
+utl.md
+Names: 1,        Desc: 1,        Syn: 1,        Options: 1,        Args: 1
+Processing ../src/ant/messages.txt
+Info: 2, Warn: 4, Error: 3
```

Comment:
Is this going to need updating every time we add a new message?

Reply:
Yes.

Reply:
That is too much overhead.  Can you setup a more local test?

Reply:
Just to clarify, there is also the `man-to-tcl` check that needs to be updated when Tcl code is updated (when they add a new flag/key for eg.)

I will add in 2 tests for each module, correct?

Reply:
@maliberty I have added 2 new local tests to `src/ant`, and modified the `docs/src/test` to be depending on static files instead. Could you have a look?
`ant_man_tcl_check` checks for man,proc, readme counts
`ant_readme_msgs_check` checks for readme counts and messages count.